qa/L0_optuna_bls_model/check_results.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+# Copyright 2021-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
This isn't the latest license. You might want to copy the current license from somewhere (e.g. [here](https://github.com/triton-inference-server/client/blob/db888f1aca588a10f5e4a4b02a4e4ff60d437b6f/src/c%2B%2B/perf_analyzer/genai-perf/genai_perf/parser.py#L1-L25)).

Reply:
Might not be the latest, but this is the copyright used throughout the entire MA codebase